src/Client/src/app/shared/bids/bids.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="row mt-5">
+  <div class="col ms-5">
+    <div class="d-flex justify-content-center">
+      <img [src]="itemDetails.item.imageUrl" alt="{{ itemDetails.item.name }}" class="img-fluid">
+    </div>
+  </div>
+  <div class="ms-4 pe-5 col-md col-sm-12">
+    <h2 class="text-center">{{ label }}</h2>
+    <div class="d-flex justify-content-center">
+      <form [formGroup]="form" (ngSubmit)="onSubmitForm()" class="form-group mt-2">
+        <div class="form-control border-bottom border-1">
+          <label class="col-sm col-form-label">{{ itemDetails.item.name }}</label>
+        </div>
+        <div class="form-control border-bottom border-1">
+          <label class="col-sm col-form-label">Size: {{ size }}</label>
+        </div>
+        <div class="form-floating border-bottom border-1">
+          <input type="number" class="form-control" id="price" placeholder="Price" formControlName="price">
+          <label for="price">Price</label>
+        </div>
+        <div class="d-flex justify-content-between m-3" style="font-size: larger">
+          <div class="text-start">Fee</div>
+          <div class="text-end">+{{ fee | currency }}</div>
+        </div>
+        <div class="border-bottom border-1"></div>
+        <div class="form-control border-bottom border-1">
+          <label class="col-sm col-form-label">Total Price: {{ totalPrice | currency }}</label>
+        </div>
+        <div class="form-control">
+          <button type="submit"
+                  style="font-size: larger"
+                  class="btn btn-success w-100">
+            {{ label }}
+          </button>
```

Comment:
Przy place bid jak chcę założyć bida na niższą cenę niż lowestAsk to automatycznie daje się lowestAsk, powinno być chyba na odwrót, czyli jak złożę bida na cenę wyższą niż lowestAsk to wtedy to wtedy powinno być automatycznie lowestAsk